src/sentry_plugins/clubhouse/plugin.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.exceptions import PluginError
+from sentry.plugins.bases.issue2 import IssuePlugin2, IssueGroupActionEndpoint
+
+from sentry_plugins.base import CorePluginMixin
+from sentry_plugins.utils import get_secret_field_config
+
+from .client import ClubhouseClient
+
+
+class ClubhousePlugin(CorePluginMixin, IssuePlugin2):
+    # TODO: There currently isn't a good "list all story titles" endpoint for Clubhouse.
```

Comment:
@miketheman I've logged this with Clubhouse.

Reply:
@eliperkins Thanks - I might be able to use that endpoint, I haven't fully explored it yet.

Are you familiar with the "Link to existing issue" behavior? It can be seen with the GitHub/GitLab/Asana integrations, where the user can type in either an issue ID, or some text of the title, and the browser will perform XHR requests to Sentry server, which in turn will run some code against the 3rd party's search mechanism.

I'll try to take another pass at this sometime soon.

Some relevant sections from the GitHub plugin, that will help me find this later:

- https://github.com/getsentry/sentry-plugins/blob/5ea4fbff894bc43d229a3039fb936a2d10eeedcd/src/sentry_plugins/github/plugin.py#L71-L79
- https://github.com/getsentry/sentry-plugins/blob/5ea4fbff894bc43d229a3039fb936a2d10eeedcd/src/sentry_plugins/github/plugin.py#L108-L129
- https://github.com/getsentry/sentry-plugins/blob/5ea4fbff894bc43d229a3039fb936a2d10eeedcd/src/sentry_plugins/github/plugin.py#L197-L220

Reply:
Ah, gotcha! Thanks for the link to more info.

One more alternative I could think of would be to use the [`/projects/:id/stories`](https://clubhouse.io/api/rest/v2/#List-Stories) endpoint, since this implementation does link a project ID to the integration currently.

The `/search/stories` endpoint is definitely more flexible and possibly more relevant to the user, as it does ranking based on term frequency, age decay, completion, etc., so it may be a better fit here.

Reply:
@eliperkins I'll give it a shot, and see what I come up with.

Reply:
@eliperkins The Stories search docs could use some clarity. Is there a good place to discuss that?